rules/community/fleet/fleet_bad_action.py

```diff
@@ -0,0 +1,26 @@
+from stream_alert.shared.rule import rule
+from stream_alert.shared.rule import disable
+from publishers.community.slack.slack_layout import Summary, AttachRuleInfo \
+        , AttachPublication, AttachFullRecord
+
+@disable
```

Comment:
Can you add a comment about to remove `disable` decorator if user want to use this rule?

Reply:
I have removed this and will raise another PR with some docs as I think it's probably going to be more fitting.

Reply:
Ignore comment about PR this was intended for decorators.